docs/content/3.manual/manual.yml

```diff
@@ -271,17 +271,24 @@ sections:
             input: '"Hello, world!"'
             output: ['"Hello, world!"']
 
-      - title: "`.foo`, `.foo.bar`"
+      - title: "Object Identifier-Index: `.foo`, `.foo.bar`"
```

Comment:
Then no change is needed. My only point of criticism here was on the name `Object Identifier-Index:`